resources/views/content/SocialMedia.twig

```diff
@@ -28,6 +28,7 @@
       data-url='{{ socialUrl }}'
       data-media-url='{{ socialMediaUrl }}'
     {% endif %}
-    data-theme='{{ socialTheme }}'>
+    data-theme='{{ socialTheme }}'
+    data-lang='{{ services.sessionStorage.getLang() }}'>
```

Comment:
wenn dieser Service genutzt wird, sollte dann auch eine Abhängigkeit nach IO in plugin.json?